packages/suite/src/components/onboarding/index.ts

```diff
@@ -1,30 +1,37 @@
 import Option from './Option';
+import NeueOption, { OptionsWrapper, OptionWrapper, OptionsDivider } from './NeueOption';
 import Text from './Text';
 
 import ControlsWrapper from './Wrapper/ControlsWrapper';
-import OptionsWrapper from './Wrapper/OptionsWrapper';
+// import OptionsWrapper from './Wrapper/OptionsWrapper';
```

Comment:
This can be removed (and the file too if its not used elsewhere).

Reply:
That reminds me that after I switch all separate flows for backup/recovery/fw update to the new design I will need to really go through these old components are remove what is not used anymore. But that's something for another PR.